data/registry/instrumentation-go-go-resty.yml

```
@@ -0,0 +1,13 @@
+title: Go-Resty Instrumentation
+registryType: instrumentation
+isThirdParty: true
+language: go
+tags:
+  - go
+  - instrumentation
+  - http
+repo: https://github.com/dubonzi/otelresty
+license: Apache 2.0
+description: Custom instrumentation for the `go-resty` http client library.
```

Linter is complaining about this:

```suggestion
description: Custom instrumentation for the `go-resty` HTTP client library.
```